### DISCUSSION TITLE
Run Sophiascript from unprivileged User Account but in a elevated Shell

### DISCUSSION THREAD

**Discussion by djusHa** [farag2/Sophia-Script-for-Windows#255] [Q&A] [answered]
Is this possible?

Have trouble to start Sophiascript in a elevated Shell, there is a Message in PS Console "This User have no Admin rights" or so.

What i'm doing wrong?

**Comment by farag2** [comment, marked as answer]
This is true if your current user doesn't have an elevated priveleges, and when you want to open any app as admin, you type in your admin credentials. Am I right? :)

**Comment by farag2** [comment]
Why there is such checking. All because if you try to run such script within your account without admin rights (by elevating them with UAC prompt), you cannot configure OS due to PowerShell limitations. This is how Windows works.

**Reply by djusHa** [reply]
ok, any chance to bypass this?

**Reply by farag2** [reply]
Of course, no...

**Reply by djusHa** [reply]
Uh, really sad :-(

**Reply by farag2** [reply]
I understand you, but the script will have to be totally re-written to make it work for such situatians. Plus, I need many tests. It doesn't worth it. It has already 12 500 lines of code (in the future 5.12.4 release). I'm scary of how much work I have to do to make it work.

**Reply by djusHa** [reply]
Never mind, thanks for your great Work!

**Reply by farag2** [reply]
Unfortunately, even SophiApp (over next project, you may download it and test) relies on the same paradigm.
